src/test/java/com/thebois/models/beings/ColonyTests.java

```diff
@@ -49,4 +61,98 @@ public void updateCallsUpdateOnAllBeings() {
         }
     }
 
+    @ParameterizedTest
+    @MethodSource("getItemTypes")
+    public void emptyColonyInventoryIsEmpty(final ItemType itemType) {
+        // Arrange
+        final int beingCount = 1;
+        final Collection<IBeing> pawns = new ArrayList<>(beingCount);
+        for (int i = 0; i < beingCount; i++) {
+            pawns.add(Mockito.mock(IBeing.class));
+        }
+        final Colony colony = new Colony(pawns);
```

Comment:
DRY

Since this is used in multiple tests, you could break it out into a `mockColony(beingCount)` method.

Reply:
question

And I don't see the reason why you're mocking pawns when you don't need them? Couldn't you pass in just `List.of()`?

Reply:
Yes, made mock method. Much better 👍